app/controllers/location_controller.rb

```diff
@@ -800,7 +784,6 @@ def edit_location_description
         if (params[:delete_after] == "true") &&
            (old_desc = LocationDescription.safe_find(params[:old_desc_id]))
           v = @description.versions.latest
-          v.merge_source_id = old_desc.versions.latest.id
           v.save
```

Comment:
It doesn't look like "v" is used anywhere in this method any more.  I think you can get rid of "v = @description.versions.latest" and "v.save", too.